main.js

```diff
@@ -57,7 +64,12 @@ document.addEventListener('keydown', function (event) {
 })
 
 document.addEventListener('keydown', function (event) {
-  if (event.keyCode === 32) {
+  if (event.keyCode === 32 && moving === false) {
```

Comment:
What if you could ask your car if it `isStarted()`?

Reply:
Im not sure if what i did was what you were talking about here.